diagnosticsModule/Private/AdfsHealthChecks.ps1

```diff
@@ -390,19 +391,20 @@ Function TestADFSDuplicateSPN
                 return $testResult;
             }
         }
-        elseif ($adfsServiceAccount.Contains("LocalSystem"))
+        elseif ($adfsServiceAccount.Contains("LocalSystem") -or $adfsServiceAccount.Contains("NT AUTHORITY\NETWORK SERVICE"))
```

Comment:
We should make these checks case insensitive

Reply:
-eq is case insensitive.